ibis/expr/datatypes.py

```diff
@@ -239,11 +230,16 @@ def __repr__(self):
 
 
 class SignedInteger(Integer):
-    pass
+
+    def largest(self):
```

Comment:
Any reason not to make these `@property`s?

Reply:
No particular. When should I prefer a property over a method?